lesson_15/javyenWares-WebServer/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "javyenwares-webserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  
+  "type": "module",
+    "scripts": {
```

Comment:
These scripts don't look the way I'd expect. I think you haven't followed the correct setup instructions. Re-read the articles and see if you can figure out what step you may have missed (particularly the second article).

Reply:
Ok will do.